examples/bri-3/test/workgroupAdministration.e2e-spec.ts

```diff
@@ -0,0 +1,157 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from '../src/app.module';
+import { ethers } from 'ethers';
+
+jest.setTimeout(20000);
+
+describe('Workgroup administration', () => {
+  let app: INestApplication;
+
+  beforeEach(async () => {
+    const moduleFixture: TestingModule = await Test.createTestingModule({
+      imports: [AppModule],
+    }).compile();
+
+    app = moduleFixture.createNestApplication();
+    await app.init();
+  });
+
+  it('Logs in an internal Bpi Subject, creates two Bpi Subjects and a Workgroup and adds the created Bpi Subjects as participants to the Workgroup', async () => {
+    const accessToken = await loginAsInternalBpiSubjectAndReturnAnAccessToken(
+      app,
+    );
+
+    const createdBpiSubject1Id = await createABpiSubjectAndReturnId(
+      'External Bpi Subject 1',
+      app,
+      accessToken,
+    );
+    const createdBpiSubject2Id = await createABpiSubjectAndReturnId(
+      'External Bpi Subject 2',
+      app,
+      accessToken,
+    );
+
+    const createdWorkgroupId = await createAWorkgroupAndReturnId(
+      app,
+      accessToken,
+    );
+
+    await updateWorkgroupAdminsAndParticipants(
+      createdWorkgroupId,
+      [createdBpiSubject1Id],
+      [createdBpiSubject1Id, createdBpiSubject2Id],
+      app,
+      accessToken,
+    );
+
+    const resultWorkgroup = await fetchWorkgroup(
+      createdWorkgroupId,
+      app,
+      accessToken,
+    );
+
+    expect(resultWorkgroup.participants.length).toBe(2);
+    expect(resultWorkgroup.participants[0].id).toEqual(createdBpiSubject1Id);
+    expect(resultWorkgroup.participants[1].id).toEqual(createdBpiSubject2Id);
+  });
+});
+
+async function loginAsInternalBpiSubjectAndReturnAnAccessToken(
+  app: INestApplication,
+): Promise<string> {
+  // These two values must be inline with the value for the bpiAdmin from seed.ts
+  const internalBpiSubjectPublicKey =
```

Comment:
can we read from env var?
can we write e2e test to be executed against hardhat by forking goerli (or any other) network?

Reply:
It is the value we use in the seed script. As long as we agree that this is our seed testing data with a big disclaimer i do not see a reason to put it into variables.